### PR TITLE
80 derivation path

### PR DESCRIPTION
**Pull request by bachstatter**
Issue #80 
I might need some help to test this feature. Don't know if you have any tricks for faking Ledger logins? I guess the best way would be to test with the real thing. 

I tried to follow styles for dark mode. Also, I don't know how picky we want to be with the dropdown indicator icon, let me know if I should change it to a small triangle instead of the default caret down. 

Screen Shot 2019-11-19 at 8 14 27 pm
Screen Shot 2019-11-19 at 8 14 15 pm

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ module.exports = {
 		'no-debugger': 'off',
 		'import/no-unresolved': 2,
 		'no-undef': 2,
-		quotes: [2, 'single'],
```
Comment (clementbalestrat) [inline .eslintrc.js]: @bachstatter did you do that intentionally? 

Screen Shot 2019-11-20 at 9 11 57 am

Comment (bachstatter) [reply .eslintrc.js]: Restart webpack, 

> Let the prettier rule take care of quote linting.
> The eslint one didn't like wrapping single qoutes with double qoutes.
>Eg. "my name is 'jocke'"

Comment (bachstatter) [reply .eslintrc.js]: Prettiers linting/ fixing made more sense

Comment (clementbalestrat) [reply .eslintrc.js]: cool!

**Pull request merged**